otherlibs/action-plugin/src/sexpable_intf.ml

```diff
@@ -1,9 +1,44 @@
-open Stdune
+open Import
+
+module Deserialization_error = struct
```

Comment:
The word "deserialization" seems weird in this context. Perhaps, `Sexp_conversion_error` would be more suitable?

Reply:
It includes the `Version_mismatch` constructor though, and that's not really a conversion error from Sexp.

Reply:
Maybe just `Error` then? Not really important though.

Reply:
Yeah, that works.